0074

```diff
@@ -0,0 +1,164 @@
+Combined with other parameters like uptime (Heartbeats) and backhaul (Speed Test), it allows for more fair rewards and incentivizes the deployment of Radios at optimal locations.
+
+
+# Motivation
+
+Building the MOBILE Network began with the Genesis Period, during which 5G Hotspot owners with Radios had to send just one Heartbeat in 24 hours to prove they were online. This was a necessary starting point to kickstart the deployments. However, from the start, the goal was to introduce more and more data sources to evaluate the certainty and usefulness of coverage.
+
+To be successful, any Network must be reliable, always available, and meet the expectations of its users. To move one step closer to the Proof-of-Coverage verified by external sources, we propose to use Modeled Coverage data to evaluate performance of Radios.
+
+
+# Stakeholders
+
+This HIP affects only MOBILE SubDAO, particularly 5G Hotspot owners with Radios and users of the MOBILE Network.
+
+5G Hotspot owners: 5G Hotspot owners with Outdoor Radios will need to meet new quality coverage requirements to earn maximum MOBILE rewards. Some owners might need to adjust the locations and/or angles of their Outdoor Radios to optimize for coverage. It is possible that in some places, it won't be possible to earn max rewards due to environmental obstacles that are not possible to mitigate. This HIP also proposes to consider rewarding those deployers who were among the first to provide quality coverage in a given Hex.
+
+There will be three possible outcomes for owners of Radios:
+<ol>
+<li>Increase in earnings. Because some Radios on the MOBILE Network won't be able to meet minimum quality requirements, part of the rewards will be re-distributed proportionally to those who qualify based on all PoC criteria.</li>
+<li>Earnings will stay the same.</li>
+<li>Less or no earnings.</li>
+</ol>
+
+Hotspot owners will be able to evaluate their performance with the Modeled Coverage map. Depending on the location and time of deployment in the particular hex, there might be an opportunity to improve coverage and earn more. Changing the location and angle of the Radio requires resubmission of the CPI registration. In addition to compliance, this registration ensures new data is used for Modeled Coverage data calculations.
+
+More information about the proposed reward changes is in the Detailed Explanation section below.
+
+
+# Detailed Explanation
+
+This HIP replaces the current algorithm for MOBILE rewards that was initially introduced during the Genesis phase and was based on Radio Type multipliers with a new algorithm that uses the location of the radio to calculate rewards based on the hexes the radio actually covers, as modeled by the obstruction data oracle. Below is a detailed explanation of the proposed rewards algorithm and new requirements for scaling MOBILE rewards based on the Modeled Coverage data.
+
+## Reward Mechanics
+
+### Hex Size
+
```

Comment:
I would appreciate some more background here...how are we using Hexs today (just a sentence of 2) before we go into the change.

Reply:
@meowshka

Reply:
It's in the second paragraph, let me know if it needs more explanation. 

This HIP introduces a fundamental difference in the use of hexes for Mobile Network compared to IoT. In IoT, hexes are used to determine the locations of the Hotspots and the number of Hotspots in the given hex. On the contrary, in Mobile Network, hexes are used to determine where the coverage exists. The Radio doesn't have to be in a given hex to provide coverage there. The use of Modeled Coverage data allows us to un-tie the device's location from its signal strength and propagation. To sum up, the hexes in Mobile Network are used to identify the level of coverage in a particular hex from all the different radios. A Radio can earn rewards by providing coverage in multiple hexes.